bin/reth/tests/commands/bitfinity_node_it.rs

```diff
@@ -169,23 +174,64 @@ async fn bitfinity_test_node_forward_send_raw_transaction_requests() {
     // Arrange
     let _log = init_logs();
 
-    let eth_server = EthImpl::new();
+    let eth_server = EthImpl::new().await;
     let (_server, eth_server_address) =
         mock_eth_server_start(EthServer::into_rpc(eth_server)).await;
     let (reth_client, _reth_node) =
         start_reth_node(Some(format!("http://{}", eth_server_address)), None).await;
 
     // Create a random transaction
-    let mut tx = [0u8; 256];
-    rand::thread_rng().fill_bytes(&mut tx);
-    let expected_tx_hash =
-        keccak::keccak_hash(format!("0x{}", reth_primitives::hex::encode(tx)).as_bytes());
+    let tx = transaction_with_gas_price(100);
+    let encoded = alloy_rlp::encode(&tx);
+    let expected_tx_hash = keccak::keccak_hash(&encoded);
 
     // Act
-    let result = reth_client.send_raw_transaction_bytes(&tx).await;
+    let result = reth_client.send_raw_transaction_bytes(&encoded).await.unwrap();
 
     // Assert
-    assert_eq!(result.unwrap(), expected_tx_hash.0);
+    assert_eq!(result.to_fixed_bytes(), expected_tx_hash.0.to_fixed_bytes());
+
+    tokio::time::sleep(Duration::from_secs(1)).await;
```

Comment:
do not sleep. Replace the TXS_ORDER queue with a tokio channel and read from it

Reply:
Done